src/main.rs

```diff
@@ -177,6 +185,55 @@ async fn stop(ctx: &Context, msg: &Message) -> CommandResult {
     Ok(())
 }
 
+#[command]
+#[only_in(guilds)]
+async fn skip(ctx: &Context, msg: &Message, _args: Args) -> CommandResult {
+    let guild = msg.guild(&ctx.cache).unwrap();
+    let guild_id = guild.id;
+
+    let manager = songbird::get(ctx)
+        .await
+        .expect("Songbird Voice client placed in at initialisation.")
+        .clone();
+
+    if let Some(handler_lock) = manager.get(guild_id) {
+        let handler = handler_lock.lock().await;
+        let queue = handler.queue();
+        let curr = queue.current();
+
+        // @TODO: skip doesn't work as well as manual de-queueing
+        // let _skip = queue.skip();
```

Comment:
I'd say its fair that we research this issue futher before finalising this feature and commiting to this "workaround".